timetracker/entries/models.py

```diff
@@ -2,13 +2,20 @@
 
 from django.utils import timezone
 
+class Client(models.Model):
+    """You may work with different clients for same/different projects,
+    hence let's create a model for client  """
+    name = models.CharField(max_length=200)
+
+    def __str__(self):
+        return '{}'.format(self.name)
 
 class Project(models.Model):
-    client = models.CharField(max_length=200)
     name = models.CharField(max_length=200)
+    client_id = models.ForeignKey('Client', null=True)
```

Comment:
You could add another `AlterField` migration at the end of your migration to get this back to just `client`.

If you look at your database, you'll have a field called `client_id_id` (which is a little icky).